testdata/testdata.go

```diff
@@ -246,6 +246,8 @@ func GenTestData(mem memory.Allocator, sc *arrow.Schema, opts GenTestDataOptions
 	return records
 }
 
+// GenTestDataV1 does approximately the same job as GenTestData, however, it's intended for simpler use-cases.
+// Deprecated. Will be removed in future release.
```

Comment:
```suggestion
// Deprecated: Will be removed in future release.
```

According to https://go.dev/blog/godoc